### DISCUSSION TITLE
DONE: Option to reset Round Robin to active window after inactivity or right/left click

### DISCUSSION THREAD

**Discussion by I3igAl** [WowOpenBox/WowOpenBox#86] [Ideas]
Very often I have seen round robin stop midway through the rotation, say on window 4, after a fight. when i go to start the next fight, i use window 1 with my tank character to click and choose the next target. I expect it to start with window 1 but it starts from 4. 

I would like to see round robin reset to whichever window is the active one, be it after a certain amount of time that round robin does not get activated, or as soon as a click action is targeted at a window. I am aware that Control Shift # will set the focus but that is a cumbersome keybind to press every time to ensure round robin starts where i want it to.

While typing this I realized that clicked a window mid fight and changing the round robin might be clunky, so maybe just the timed out option.

**Comment by I3igAl** [comment]
After messing around in the Raw Settings file I discovered hk,focusMain Ctrl-Shift-W wich defaults the focus to 1. I also rebound it to Shift Alt W which is more comfortable for me so i guess my suggestion is not needed.

**Comment by mooreatv** [comment]
it's actually a good idea and semi duplicate of #84 that I implemented in 3.8
(focusmain hotkey should be in the help too (ctrl-shift-w) and it focuses not just wob1 but whichever window is currently in slot 1)

keep ideas coming!
